### PR TITLE
Add basic tests for profiles

### PR DESCRIPTION
**Pull request by igordavydsson**
Basic tests + travis setup.

Refs #1

### CODE REVIEW

**Comment by igordavydsson** [comment]
or maybe @tituomin could roll Travis for us?

**Comment by tituomin** [comment]
I activated Travis. Maybe push something to start the build?

**Comment by igordavydsson** [comment]
@tituomin thanks! I will fix the problems :)

File: profiles/tests/test_profiles_api.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    assert Profile.objects.count() == 0
+
+    post_create(user_api_client, PROFILE_URL)
+
+    assert Profile.objects.count() == 1
+    profile = Profile.objects.latest('id')
+    assert profile.user == user_api_client.user
+
+
+def test_post_update_own_profile(user_api_client, profile):
```
Comment (tuomas777) [inline profiles/tests/test_profiles_api.py]: maybe `test_put_update_own_profile` ?

File: profiles/tests/test_profiles_api.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    assert Profile.objects.count() == 1
+
+    user_profile_url = get_user_profile_url(profile)
+    phone_number_data = {'phone': '0461234567'}
+
+    put_update(user_api_client, user_profile_url, phone_number_data)
+
+    updated_profile = Profile.objects.get(user=profile.user)
```
Comment (tuomas777) [inline profiles/tests/test_profiles_api.py]: You could use `profile.refresh_from_db()` which would also make sure, that we are dealing with the same profile instance.

File: profiles/tests/test_profiles_api.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    assert updated_profile.phone == phone_number_data['phone']
```
Comment (tuomas777) [inline profiles/tests/test_profiles_api.py]: What happens to the other fields? If we want to comply with RFC 2616, they should be emptied I guess.

Comment (igordavydsson) [reply profiles/tests/test_profiles_api.py]: so far other fiealds are empty 😬 
but good point, we'll check that it works that way in other cases. 👍

File: profiles/tests/test_profiles_api.py (hunk 2 above)
Comment (igordavydsson) [reply profiles/tests/test_profiles_api.py]: nice 👍

File: profiles/tests/test_profiles_api.py (hunk 1 above)
Comment (igordavydsson) [reply profiles/tests/test_profiles_api.py]: 👍

**Comment by igordavydsson** [comment]
@tituomin @tuomas777 does it look ok to be merged?

**Pull request merged**